module/zcommon/zfeature_common.c

```diff
@@ -553,6 +553,13 @@ zpool_feature_init(void)
 	    "com.datto:resilver_defer", "resilver_defer",
 	    "Support for defering new resilvers when one is already running.",
 	    ZFEATURE_FLAG_READONLY_COMPAT, ZFEATURE_TYPE_BOOLEAN, NULL);
+
+	{
+	zfeature_register(SPA_FEATURE_HEAD_ERRLOG,
+	    "com.delphix:head_errlog", "head_errlog",
+	    "Improved zpool status command output.",
```

Comment:
The feature is actually the logging of the errors to the ZAP, so maybe:
"Introduce an on-disk block error log"